### DISCUSSION TITLE
Is it possible to resume Mlflow run?

### DISCUSSION THREAD

**Discussion by kondera** [pycaret/pycaret#4012] [General]
Hey, 

Is it possible to resume mlflow run? I've compared some models, but ended experiment to work on something else. Now I would like to attach to this run and tune some models. 
I also tried looking for way to set run name (not really helpful to have names like Session initialized XXX), but couldn't find any info about that.

**Comment by celestinoxp** [comment]
You must save your experiment or trained model... see here:
https://pycaret.readthedocs.io/en/latest/api/classification.html#pycaret.classification.save_experiment
or
https://pycaret.readthedocs.io/en/latest/api/classification.html#pycaret.classification.save_model

**Reply by kondera** [reply]
Well yeah, but I don't really need to save experiment. It wouldbe nice if  there's possibility to attach to a run in mlflow experiment. I don't really need to save the models locally, because I can just logged them into mlflow. I tried setting mlflow run id env, but it throws errors.
